odxtools/table.py

```diff
@@ -114,16 +135,21 @@ def _get_common_props(et_element):
         description=description,
     )
 
+
 def read_table_row_from_odx(et_element):
     """Reads a TABLE-ROW."""
-    key = et_element.findtext("KEY")
-    structure_ref = ""
+    key = et_element.find("KEY").text
```

Comment:
any reason why you changed this? `findtext()` seems to be a little bit more robust, because if there is no `KEY` tag, the new code will throw an exception.

Reply:
There is no specific reason other than being consistent with the rest of the codebase. The previous MR updated only one occurrence of this code snippet. I can do another short MR to simply search and replace?

<img width="1132" alt="image" src="https://user-images.githubusercontent.com/5611865/185660414-2e23e0e1-5bf7-4aa0-b313-3465ce8c27d9.png">

Reply:
Created an issue for findtext(). I'll follow up with another MR. 

https://github.com/mercedes-benz/odxtools/issues/58

Reply:
I suppose `findtext()` ought to be used whereever it makes sense. I have to confess that it is a recent find for me as well, so there are probably a few "legacy" places...